packages/babel-plugin-emotion/src/utils/transform-expression-with-styles.js

```diff
@@ -123,6 +124,14 @@ export let transformExpressionWithStyles = ({
 
       return { node, isPure: true }
     }
+
+    if (canAppendStrings && shouldLabel) {
+      const label = getLabelFromPath(path, state, t)
+      if (label) {
+        appendStringToArguments(path, `;label:${label};`, t)
```

Comment:
Wasn't 100% sure yet if avoiding trailing semi in here would also be OK. I plan this to be dev-only though so it doesn't quite matter.